extension/src/commands/external.ts

```diff
@@ -11,6 +11,14 @@ export enum RegisteredCliCommands {
   EXPERIMENT_RESET_AND_RUN = 'dvc.resetAndRunCheckpointExperiment',
   QUEUE_EXPERIMENT = 'dvc.queueExperiment',
 
+  EXPERIMENT_TREE_APPLY = 'dvc.views.experimentsTree.applyExperiment',
+  EXPERIMENT_TREE_BRANCH = 'dvc.views.experimentsTree.branchExperiment',
+  EXPERIMENT_TREE_QUEUE = 'dvc.views.experimentsTree.queueExperiment',
+  EXPERIMENT_TREE_REMOVE = 'dvc.views.experimentsTree.removeExperiment',
+  EXPERIMENT_TREE_RESUME = 'dvc.views.experimentsTree.resumeCheckpointExperiment',
+  EXPERIMENT_TREE_RUN = 'dvc.views.experimentsTree.runExperiment',
+  EXPERIMENT_TREE_RESET_AND_RUN = 'dvc.views.experimentsTree.resetAndRunCheckpointExperiment',
```

Comment:
These are problematic. We need to call the CLI from inside experiments after we get the params. There needs to be two different behaviours. One for the runner and one for the executor (QUEUE)

Reply:
We want to offer to show the error when trying to Queue but not when trying to run in the runner.